src/test/kotlin/no/nav/syfo/dialogmote/GetDialogmoteEnhetApiSpek.kt

```diff
@@ -0,0 +1,165 @@
+import no.nav.syfo.testhelper.UserConstants.ENHET_NR_NO_ACCESS
+import no.nav.syfo.testhelper.UserConstants.VEILEDER_IDENT
+import no.nav.syfo.testhelper.generator.generateNewDialogmote
+import no.nav.syfo.testhelper.mock.*
+import no.nav.syfo.util.bearerHeader
+import no.nav.syfo.varsel.arbeidstaker.brukernotifikasjon.*
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class GetDialogmoteEnhetApiSpek : Spek({
+    val objectMapper: ObjectMapper = apiConsumerObjectMapper()
+
+    describe("DialogmoteApiSpek") {
+
+        with(TestApplicationEngine()) {
+            start()
+
+            val modiasyforestMock = ModiasyforestMock()
+            val syfomoteadminMock = SyfomoteadminMock()
+            val syfopersonMock = SyfopersonMock()
+            val tilgangskontrollMock = VeilederTilgangskontrollMock()
+
+            val applicationState = testAppState()
+
+            val database = TestDatabase()
+
+            val embeddedEnvironment = KafkaEnvironment(
+                autoStart = false,
+                withSchemaRegistry = false,
+                topicNames = listOf(
+                    BRUKERNOTIFIKASJON_OPPGAVE_TOPIC,
+                    BRUKERNOTIFIKASJON_DONE_TOPIC,
+                )
+            )
+
+            val environment = testEnvironment(
+                kafkaBootstrapServers = embeddedEnvironment.brokersURL,
+                modiasyforestUrl = modiasyforestMock.url,
+                syfomoteadminUrl = syfomoteadminMock.url,
+                syfopersonUrl = syfopersonMock.url,
+                syfotilgangskontrollUrl = tilgangskontrollMock.url
+            )
+
+            val brukernotifikasjonProducer = mockk<BrukernotifikasjonProducer>()
+
+            val wellKnownSelvbetjening = wellKnownSelvbetjeningMock()
+            val wellKnownVeileder = wellKnownSelvbetjeningMock()
+
+            application.apiModule(
+                applicationState = applicationState,
+                brukernotifikasjonProducer = brukernotifikasjonProducer,
+                database = database,
+                environment = environment,
+                wellKnownSelvbetjening = wellKnownSelvbetjening,
+                wellKnownVeileder = wellKnownVeileder,
+            )
+
+            justRun { brukernotifikasjonProducer.sendOppgave(any(), any()) }
+
+            afterEachTest {
+                database.dropData()
+            }
+
+            beforeGroup {
+                syfopersonMock.server.start()
+                tilgangskontrollMock.server.start()
+                embeddedEnvironment.start()
+            }
+
+            afterGroup {
+                syfopersonMock.server.stop(1L, 10L)
+                tilgangskontrollMock.server.stop(1L, 10L)
+
+                database.stop()
+                embeddedEnvironment.tearDown()
+            }
+
+            describe("Get Dialogmoter for PersonIdent") {
```

Comment:
"Get Dialogmoter for PersonIdent" -> "... for EnhetNr"?

Reply:
Fikset.